gradle/dependencies.gradle

```diff
@@ -12,7 +12,7 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '79.0.2'
+      mapboxNavigatorVersion = '79.0.3'
```

Comment:
This is unrelated to "fixing the route alternative thread issue"

But this is an extra change to prepare for the `2.1.0-rc.2` release

Reply:
Lets add this to the changelog under "Fixed a downstream issue of offline routing being slow." same as for 2.0.1.